packages/swirl-components/src/components/swirl-app-layout/swirl-app-layout-base.css

```diff
@@ -320,6 +318,37 @@
   }
 }
 
+.app-layout--content-scrollable {
+  &:not(.app-layout--content-scrolled-to-top) {
+    & .app-layout__app-bar {
+      border-bottom-color: var(--s-border-default);
+    }
+  }
+
+  &.app-layout--has-bottom-bar:not(.app-layout--content-scrolled-to-bottom) {
+    & .app-layout__content {
+      border-bottom: var(--s-border-width-default) solid var(--s-border-default);
```

Comment:
**nitpick (if-minor):** This is just something minor, but i prefer when you only change the border to transparent instead of disabling it because then the view jumps by 1 pixel every time you scroll down. You did this in the other places, maybe you can do it here too.

Reply:
Good catch. Done.